src/components/Projects/index.tsx

```diff
@@ -135,7 +136,19 @@ export default function ProjectsList(): JSX.Element {
           <div>
             <Grid container spacing={4}>
               <Grid item xs={12}>
-                {results && <Table id='projects-table' columns={columns} rows={results} orderBy='name' />}
+                {results && (
+                  <Table
+                    id='projects-table'
+                    columns={columns}
+                    rows={results}
+                    orderBy='name'
+                    Renderer={ProjectCellRenderer}
+                    showCheckbox
+                    selectedRows={[]}
+                    setSelectedRows={() => undefined}
```

Comment:
this to be set later?

Reply:
Yea... it should be. The designs don't show any sort of menu or bulk action dropdown to do anything with the selected items, so for now they are just here to make the check boxes. I'll ask in the designs if that is intentional or not.